.github/workflows/wipcheck.yml

```diff
@@ -4,7 +4,7 @@ on:
   pull_request:
     types: 
       - opened        # when PR is opened
-      - edited        # when PR title is edited
+      - edited        # when PR is edited
```

Comment:
Not only title. I noticed that editing my comment also triggers wipcheck.

Reply:
Well, it is true only for the first comment. Maybe it is treated in some special way.